package.json

```diff
@@ -42,7 +42,7 @@
     "jsdom": "^22.1.0",
     "prettier": "^2.8.7",
     "semver": "^7.6.0",
-    "typescript": "^4.9.5",
+    "typescript": "<5.1.0",
```

Comment:
why this pin? this seems bad at first glance

Reply:
I changed it because I needed Typescript 5.x.x but also was having a compatibility warning show up when running linting. Let me poke at this a bit.

Reply:
Changed this, we'll now get a warning on linting but I think that's fine.